src/helpers/bodyTrap.js

```diff
@@ -0,0 +1,48 @@
+import portalOpenInstances from "./portalOpenInstances";
+// Body focus trap see Issue #742
+
+let before, after, instances;
+
+function focusContent() {
+  if (!instances || instances.length === 0) {
+    // eslint-disable-next-line no-console
+    console.warn(`React-Modal: Open instances > 0 expected`);
+    return;
+  }
+  instances[instances.length - 1].focusContent();
```

Comment:
Maybe we can let this be no-op if there are no modals opened, or, maybe warn only in develoment mode. What do you think? 
 
```js
if (instances.lenght > 0) {
  instances[instances.length - 1].focusContent();
}
```

Reply:
@diasbruno I agree, and as per the pattern in ModalPortal.js I've made all consoles only print in dev mode.